File: pgm/ABC_Java/ABC_238_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final double n = Double.parseDouble (reader.readLine());
        final String Yes = "Yes";
        final String No = "No";
        final double two = Double.valueOf (2);
        final double left = Math.pow (two, n);
        final double right = Math.pow (n, two);
        if (Double.valueOf (left).compareTo (Double.valueOf (right)) > 0) {
            System.out.println (Yes);
        } else {
            System.out.println (No);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_238_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val dTwo: Double = 2.0

fun main() {
    val n: Double = readln().toDouble()
    val left: Double = dTwo.pow(n)
    val right: Double = n.pow(dTwo)
    if (left > right) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}